packages/expo-router/src/views/Sitemap.tsx

```diff
@@ -215,6 +217,53 @@ function ArrowIcon({ rotation = 0 }: { rotation?: number }) {
   );
 }
 
+function SystemInfo() {
+  const getLocationOrigin = () => {
+    if (Platform.OS === 'web' && typeof window !== 'undefined') {
```

Comment:
```suggestion
    if (process.env.EXPO_OS === 'web' && typeof window !== 'undefined') {
```

Also, window.location is set on all platforms in Expo Router.